piperider_cli/profiler/profiler.py

```diff
@@ -74,6 +74,7 @@ def profile(self, tables=None):
                         metadata.schema = schema
                     Table(table, metadata, autoload_with=self.engine)
 
+        result["metadata"] = metadata
```

Comment:
What information the handler would like to get? Tables and columns? Actually, i prefer not to expose the metadata because the table/columns that the profiler will profile is the internal logic of profiler. In the near future, we may introduce virtual table, the table would not be presented in the metadata.